src/Symfony/Component/Cache/Tests/Simple/FilesystemCacheTest.php

```diff
@@ -22,4 +22,52 @@ public function createSimpleCache($defaultLifetime = 0)
     {
         return new FilesystemCache('', $defaultLifetime);
     }
+
+    public function testPrune()
```

Comment:
How about this approach: https://github.com/symfony/symfony/pull/23451/files#diff-6388c149445d15d020386824fc3440c5R63